README.md

```diff
@@ -5,7 +5,23 @@
 
 #### [TANDA blog post](https://hazyresearch.github.io/snorkel/blog/tanda.html)
 
-#### Hazy Research: [GitHub](https://github.com/HazyResearch), [research homepage](https://snorkel.stanford.edu)
+#### Paper (NIPS 2017): [Learning to Compose Domain-Specific Transformations for Data Augmentation](https://arxiv.org/abs/1709.01643) 
+
+#### Hazy Research: [GitHub](https://github.com/HazyResearch), [research homepage](http://snorkel.stanford.edu)
+
+## NEW: an easy-to-use Keras interface
+
+Just in time for NIPS 2017, we're releasing an easy-to-use
+[Keras](https://keras.io/) interface for TANDA. Just train your TAN, load the
```

Comment:
Re: "Just train your TAN...": I'd say let's lead with a one line (bolded and red?) about Keras interface + pre-trained TAN.  Then mention can be used with any TAN you train / is implied

Reply:
I'd suggest something like: "Just in time for NIPS 2017, we're releasing an easy-to-use, pre-trained substitute for Keras' ImageDataGenerator data augmentation class. Just swap in `TANDAImageDataGenerator` and you'll be using our trained data augmentation generators!"